client/duckcoin.go

```diff
@@ -44,6 +45,7 @@ Examples:
    duckcoin 1 -t nSvl+K7RauJ5IagU+ID/slhDoR+435+NSLHOXzFBRmo= -a 3 -m "Payment of 3 Quacks to Ishan"`
 )
 
+// A Block is a data structure that represents a block in the blockchain.
```

Comment:
IMO this doesn’t explain much about what blocks are, I can see what you’re trying to do tho.

I made Duckcoin to learn about how crypto works, and there’s a real lack of simple implementations, so I promised I would document the Duckcoin code really well, thanks for helping me!
```suggestion
// A Block is a data structure that represents a validated set of transactions with proof of work, which makes it really hard to rewrite the blockchain.
```